packages/xo-web/src/common/intl/messages.js

```diff
@@ -1055,13 +1055,16 @@ const messages = {
   hostCpusNumber: 'Core (socket)',
   hostManufacturerinfo: 'Manufacturer info',
   hostBiosinfo: 'BIOS info',
+  hostUsbs: 'USBs',
```

Comment:
Don't add a prefix, this can be used in other place than inside an host.
`usbs: USBs`